platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -161,14 +173,29 @@ BundleContext getBundleContext() {
         return warnings;
     }
 
+    private Collection<Describable> listMigratableDescribables() {
```

Comment:
Why was that not an option for the actual export calls? I followed the same technique already there.